site/content/docs/cli/constructs-reference.md

```diff
@@ -271,6 +271,32 @@ new BrowserCheck('browser-check-1', {
 - `code`: an object with either an `entrypoint` property that points to `.spec.js|ts` file, or a `content` property with
 raw JavaScript / TypeScript as a string.
 
+## `MultiStepCheck`
+
+As Browser Checks, Multistep API checks are based on [`@playwright/test`](https://playwright.dev/). You can just write `.spec.js|ts` files with test cases
```

Comment:
This is not accurate.
MultiStep checks needs to be defined first on a construct and the using `code.entrypoint` a `.spec.ts` file can be assigned to it.

By default if there's any `.spec.ts` without a construct is assumed as browser check